ncm-dpmlfc/src/main/pan/components/dpmlfc/schema.pan

```diff
@@ -125,11 +147,53 @@ type ${project.artifactId}_component_dpns_node_config = {
         "readonly" ? boolean
 };
 
+# DAV related parameters: see zlcgdm-dav.conf for parameter documentation
```

Comment:
Because it has to be coordinated with other DPM related config and services. Apache is used but with a specific module that has its own configuration. This is what is managed by this config module.